spec/features/agents/admin_can_configure_agent_access_rights_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+describe "Admin can configure the organisation" do
+  let(:territory) { create(:territory) }
+  let(:organisation) { create(:organisation, territory: territory) }
+
+  let(:agent) { create(:agent, basic_role_in_organisations: [organisation], role_in_territories: [territory]) }
+  let!(:agent_access_right) { create(:agent_territorial_access_right, allow_to_manage_access_rights: true, territory: territory, agent: agent) }
+  let(:other_agent) { create(:agent, basic_role_in_organisations: [organisation], role_in_territories: []) }
+  let!(:other_agent_access_right) { create(:agent_territorial_access_right, allow_to_manage_access_rights: false, territory: territory, agent: other_agent) }
+
+  before do
+    login_as(agent, scope: :agent)
+    visit edit_admin_territory_agent_path(territory, other_agent)
+  end
+
+  it "can give territorial admin access to other agent" do
+    check("Administrateur du territoire")
+    within(".agent-territorial") do
+      click_on("Enregistrer")
+    end
+    expect(other_agent.reload.territorial_admin_in?(territory)).to eq(true)
```

Comment:
Pour la lisibilité et la sécurité du test, il est préférable d'utiliser le matcher `change`. :wink: 

```suggestion
    within(".agent-territorial") do
      expect { click_on("Enregistrer") }.to change { other_agent.reload.territorial_admin_in?(territory) }.to(true)
    end
```

Reply:
Pour la lisibilité, je ne suis pas convaincu et pour la sécurité, je ne suis pas sûr de suivre. Mais ta proposition me convient.

Merci.

Reply:
Par sécurité, je veux dire que le test doit vérifier que c'est bien le clic sur le bouton qui change le statut admin. Si on vérifie seulement le statut après avoir cliqué sur le bouton, on ne sait pas si le statut a changé, ou si c'était à `true` même avant de cliquer le bouton ! :wink:

Reply:
ok, c'est plus clair. Merci